include/net/lwm2m.h

```diff
@@ -136,6 +136,11 @@ struct lwm2m_ctx {
 	 *  copied into the actual resource buffer.
```

Comment:
Technically, such commit's should contain `[nrf toup]` sauce tag, such approach is now "forbidden due to excessive abuse" (see development model page on confluence). You should rather open an upstream PR first, and then cherrypick the commits with `[nrf fromlist]`.